.circleci/config.yml

```diff
@@ -978,6 +978,7 @@ workflows:
                 - main
                 - chore/update-tsconfig-for-node-20
                 - update-functional-test-executor
+                - update-ci-image
```

Comment:
This is interesting. When do these branch filters get cleaned-up?

Reply:
Good question. I am not sure as the [ecosystem doc](https://mozilla.github.io/ecosystem-platform/how-tos/ci-guidelines#updating-ci-runner-images) doesn't mentions anything about removing but just adding which is what I was following to update the CI runner. But IMO we can remove the old branches after the patch lands, I think.

Reply:
I will confirm on this and remove the old branches.

Reply:
The old branches have been removed.